ext/crates/fp/src/vector_inner.rs

```diff
@@ -488,10 +485,16 @@ impl<'a, const P: u32> SliceP<'a, P> {
     }
 
     #[inline]
-    fn limb_range(&self) -> (usize, usize) {
+    fn limb_range(&self) -> Range<usize> {
         limb::range::<P>(self.start, self.end)
     }
 
+    #[inline]
+    fn limb_range_inner(&self) -> Range<usize> {
+        let range = self.limb_range();
+        (range.start + 1)..(usize::max(range.start + 1, range.end - 1))
```

Comment:
I think this can still underflow, which panics under debug mode. I'm happy to add a comment in the docs and leave it, since this will be pretty obvious if it happens.